src/pydap/tests/test_responses_ascii.py

```diff
@@ -53,24 +51,25 @@ def test_headers(self):
 
     def test_body(self):
         """Test the generated ASCII response."""
-        self.assertEqual(self.res.text, """Dataset {
-    Sequence {
-        String id;
-        Int32 lon;
-        Int32 lat;
-        Int32 depth;
-        Int32 time;
-        Int32 temperature;
-        Int32 salinity;
-        Int32 pressure;
-    } cast;
-} SimpleSequence;
----------------------------------------------
-cast.id, cast.lon, cast.lat, cast.depth, cast.time, cast.temperature, cast.salinity, cast.pressure
-"1", 100, -10, 0, -1, 21, 35, 0
-"2", 200, 10, 500, 1, 15, 35, 100
-
-""")
+        self.assertEqual(self.res.text,
```

Comment:
Yes, this bugs me to have the explicit EOLs and all of the extra padding at the beginning. Python's multiline strings exist for a reason :) I think that if you revert these comparisons back to checking against a multiline string (preferred), or just turn off flake8 checking on this file, then I'll be happy and we can merge this.